config/development.js

```diff
@@ -50,6 +50,47 @@ module.exports = {
       "-----END RSA PRIVATE KEY-----",
   },
 
+  botToken: {
+    publicKey:
+      "-----BEGIN PUBLIC KEY-----\n" +
+      "MIIBITANBgkqhkiG9w0BAQEFAAOCAQ4AMIIBCQKCAQBK3CkprcpAYxme7vtdjpWO\n" +
+      "gFFjoYsqU3OmhMEty/s1gnW5tgbK4ief4xk+cU+mu3YvjzWudT/SV17tAWxL4Y+G\n" +
+      "incJwL5gpQwlnw9qOAdRGkpBriQLec7kNVIydZXbUitziy+iSimxNzdDmjvlK9ZG\n" +
+      "miVLZm+MePbUtgaIpfgd+4bRWzudlITiNmWY7HppLzyBw+037iEICM4kwPPFI+SO\n" +
+      "GJhpAAmD6vk0MeZk1NeQmyQp/uOPpWmVRzgyK+XVc6AwZHV+/n6xAIT91/DjJlD1\n" +
+      "N+nS7Sqo3RJ04+KlNRUclzINOC7JBYkKtG7YQ0U9nNLkRrRlON+O6tY4OT86T1O1\n" +
+      "AgMBAAE=\n" +
+      "-----END PUBLIC KEY-----",
+    privateKey:
```

Comment:
We should not give a private key over here as our backend does not require it.

Reply:
We need it for testing purposes. Though I have removed the private key from here as it's not needed here but in `test.js` config